storageData.js

```diff
@@ -0,0 +1,34 @@
+const nameInput = document.getElementById('username');
+const emailInput = document.getElementById('email');
+const textAreaBoxInput = document.getElementById('error-message');
+const formStorage = document.getElementById('form')
+
+
+// Parsing string from local storage
+const userInfo = JSON.parse(localStorage.getItem('userInfo'))
+
+
+if (userInfo) {
+    nameInput.value = userInfo.username;
+    emailInput.value = userInfo.email;
+    textAreaBoxInput.value = userInfo.textAreaBox;
+}
+
+
+
+document.getElementById('form').addEventListener('submit', function (event) {
```

Comment:
As the previous reviewer requested, kindly ensure that entered data is saved to the Local storage even without clicking the submit button.
Please check the gifs below to understand better.

![Screen Recording 2023-01-07 at 10 45 02 PM](https://user-images.githubusercontent.com/25887253/211168100-c76c1cd4-7901-4ee6-b166-34c518758cf1.gif)
![Screen Recording 2023-01-07 at 10 45 35 PM](https://user-images.githubusercontent.com/25887253/211168104-90a571b0-f7a7-40f5-91de-ef6b62de0629.gif)